app/dashboard/sync/rsk.py

```diff
@@ -25,7 +25,6 @@ def find_txn_on_rsk_explorer(fulfillment):
         for txn in response['result']:
             if (
                 txn['from'] == funderAddress.lower() and
-                txn['to'] == payeeAddress.lower() and
                 not txn_already_used(txn['hash'], token_name)
```

Comment:
let's keep this and figure out a smarter way of ERC-20 or make it optional

Reply:
done. optional for erc20